src/sentry/web/urls.py

```diff
@@ -101,6 +101,12 @@
         api.ClientConfigView.as_view(),
         name="sentry-api-client-config",
     ),
+    # Forbidden Relay endpoint
+    url(
```

Comment:
Do we actually proxy these requests through Sentry?  Are there any cases when this also done purely on LB?

Reply:
`/*` is sent to sentry web by default, there are only few overrides, and `/api/relay/*` is not one of them. That means, they are currently handled by the redirect endpoint at the bottom of this file. 

We certainly don't want to expose the actual Relay endpoint. If we add a 404 endpoint here, we also show intent that these URLs are not meant to be used in other ways. We _could_ also handle this on LBs, but this seems like the more robust solution (also for self-hosted).